app/patients/edit/template.hbs

```diff
@@ -147,6 +147,12 @@
                     {{ext-radio content=patientTypes value=model.patientType showInline=true }}
                   </div>
                 </div>
+                <div class="form-input-group">
```

Comment:
This block should be wrapped with an if that only displays this checkbox if the patient is new.  Eg:
```handlebars
{{#if model.isNew}}
  <div class="form-input-group">
    <label>Create An Outpatient Visit</label>
    <div>
      {{em-checkbox property="createOutpatientVisit" }}
      </div>
  </div>
{{/if}}
```